src/hyperlight_host/src/sandbox/mod.rs

```diff
@@ -20,16 +20,17 @@ pub mod config;
 pub(crate) mod host_funcs;
 /// Functionality for dealing with `Sandbox`es that contain Hypervisors
 pub(crate) mod hypervisor;
-/// Functionality for dealing with initialized sandboxes that can
-/// call 0 or more guest functions
-pub mod initialized_multi_use;
 /// Functionality for dealing with memory access from the VM guest
 /// executable
 pub(crate) mod mem_access;
 /// Functionality for interacting with a sandbox's internally-stored
 /// `SandboxMemoryManager`
 pub(crate) mod mem_mgr;
 pub(crate) mod outb;
+/// Functionality for dealing with initialized sandboxes that can
+/// call 0 or more guest functions
+#[allow(clippy::module_inception)]
```

Comment:
Do we want this?

Reply:
do you mean allowing this clippy lint? Without it we either go back to the previous name, or it would be a bigger refactor.